Contributors.md

```diff
@@ -439,5 +439,29 @@
   </a>
 </td>
 <!-- End of column-36 -->
+<!-- Start of column-35 -->
```

Comment:
Hey @Eunit99 can you please remove the duplicate profile you added? 

```html
<!-- Start of column-35 -->
<td align="center">
  <a href="https://github.com/AvidCoder101">
    <img
      src="https://avatars.githubusercontent.com/u/70807684?v=4"
      width="100px"
    />
    <br />
    <sub>Avid Coder</sub>
  </a>
</td>
<!-- End of column-35 -->
```

This is what your pull request looks like. Avicoder's profile is already at number 35 so delete the one you added after 36 and push again so I can merge this. 
![duplicate](https://user-images.githubusercontent.com/62628408/158498306-d881e78a-4a3c-40d4-a390-099939456f32.jpg)

If you have any questions, feel free to ask. 🙂